homeassistant/components/light/tplink.py

```diff
@@ -27,6 +27,10 @@
 ATTR_DAILY_ENERGY_KWH = 'daily_energy_kwh'
 ATTR_MONTHLY_ENERGY_KWH = 'monthly_energy_kwh'
 
+# Need to add some non-US models
+TPLINK_KELVIN = {'LB130(US)': {'min': 2700, 'max': 9000},
+                 'LB120(US)': {'min': 2700, 'max': 6500}}
+
```

Comment:
This mapping (and a method similar to `valid_temperature_range() -> Tuple[int, int]`) should be added to `SmartBulb` class of pyhs100.

Reply:
I'm doing this !